sklearn/datasets/samples_generator.py

```diff
@@ -1196,7 +1196,7 @@ def make_sparse_spd_matrix(dim=1, alpha=0.95, norm_diag=False,
         The size of the random matrix to generate.
 
     alpha: float between 0 and 1, optional (default=0.95)
```

Comment:
It is not from your patch, but do you mind adding a space after the column here? Else numpydoc won't render it properly.